notebooks/nuextract-structure-extraction/nuextract-structure-extraction.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>&gt;&gt; To simplify the user experience, we will use <a href="https://github.com/openvinotoolkit/openvino.genai" target="_blank">OpenVINO Generate API</a> for generation of instruction-following inference pipeline.   </p><p> </p><p>I think this description is slightly misleading, can it be rephrased and simplified e.g.:</p><p> To simplify the user experience, we will use <a href="https://github.com/openvinotoolkit/openvino.genai" target="_blank">OpenVINO Generate API</a> for organization inference pipeline.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Fixed</p>